tests/openvino/excluded_tests_with_torch_export.txt

```diff
@@ -0,0 +1,504 @@
+OVModelBasicIntegrationTest::test_pipeline_2_hf_internal_testing_tiny_random_mbart
```

Comment:
Unfortunately this approach is not very robust. For example, if order of tests changes for some reason, or some test is removed, this will affect the naming. 

I also notice that for each type of test the list of failing architectures is similar, e.g. there are t5, gpt2, bart, etc. Have you considered introducing a variable like `ARCHITECTURES_FAILING_WITH_TORCH_EXPORT` containing a set of all model architectures which fail to convert with torch.export and skipping tests based on that?

Reply:
It seems to me that it should be possible to do the parsing not by the test name, but somehow through a decorator. Because from what I see, for (all?) tests one of the arguments is the architecture name. Then for all tests where you added the USE_TORCH_EXPORT argument, in theory you could also add a decorator that skips the test if USE_TORCH_EXPORT env. var. is "True" and the arguments contain an unsupported architecture name.

Reply:
I agree with @nikita-savelyevv. I missed a fact that test case id can be changed in case of removing or adding another test case.
Is it possible to slightly refactor tests to have arch name in test case name instead of their ids. If yes, we can proceed with "excluded tests" approach. It will be more convenient way because we avoid solid changes in tests and will work only with this excluded tests.
If not possible, let us proceed with Nikita's approach. For me, it is not principal. Now it is more important to start transition to torch.export().